Closes #180 blog post unix vs sas datetime

cpiraux opened the pull request:
Thank you for your Pull Request! We have developed this task checklist to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your blog post.

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `main` branch until you have checked off each task.

- [x] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update), and make sure the corresponding issue is linked in the Development section on the right hand side
- [x] Run the script from `CICD.R` line by line to first check the spelling in your post and then to make sure your code is compatible with our code-style. Address any incongruences by following the instructions in the file! 
- [x] Choose (possibly several) `tag(s)` or categories from the current list: `c("Metadata", "SDTM", "ADaM", "TLG", "Shiny", "Community", "Conferences", "Submissions", "Technical")` for your blog post. If you cannot find anything that fits your blog post, add your own tag! We occasionally tidy up all `tags` for consistency.
- [x] Add a short description for your blog post in the `description` field at the top of the markdown document.
- [x] Blog post is *short*, *personalized*, *reproducible* and *readable*
- [ ] Add a disclaimer at the top of your post, if appropriate (example: **Disclaimer** 
This blog contains opinions that are of the authors alone and do not necessarily reflect the strategy of their respective organizations.)
- [x] Address all merge conflicts and resolve appropriately
- [ ] Assign two of us (@bms63, @manciniedoardo, @StefanThoma, @kaz462) as reviewers in the PR.
- [x] Pat yourself on the back for a job well done! Much love to your accomplishment!

bms63 reviewed (commented):
Could we do something with a clock or a countdown timer for the image 
![image](https://github.com/user-attachments/assets/cd4c1b0a-1c06-42f8-a49d-a7dc9e083f53)

bms63 commented on posts/2024-07-08_unix_vs_sas_date.../unix_vs_sas_datetime.qmd:
```diff
@@ -0,0 +1,100 @@
+<!--------------- post begins here ----------------->
+
+Recent discussions within the R Consortium Submission Working Group have highlighted challenges in handling dates between SAS and Unix systems.
+In addition, during the CDISC Pilot Data update at the Phuse Test Data Factory, using R's Unix time format resulted in dates being mistakenly advanced by +10 years.
```
Could we link the Phuse Test Data Factory - https://github.com/phuse-org/TestDataFactory.  Do you mind locating and verifying the time-stamp issue.  https://rconsortium.github.io/submissions-wg/minutes/2024-06-07/  Just want to make sure this is the right one!

bms63 replied:
ACtually the meeting minutes might be nice to link as well!!

bms63 commented on posts/2024-07-08_unix_vs_sas_date.../unix_vs_sas_datetime.qmd:
```diff
@@ -0,0 +1,100 @@
+-   **R:** In contrast, R has specific data types for dates, which allow numeric dates to be rendered with a date format directly in data frames.
+    The data types in R are: "Date" for dates, "hms" or "difftime" for time, and "POSIXct" or "POSIXt" for datetime.
+
+These differences can lead to discrepancies if not properly accounted for during data conversion, potentially resulting in a date shift of 10 years when transferring data between SAS and R.
```
Could this be italicized?  It is so important!!

bms63 commented on posts/2024-07-08_unix_vs_sas_date.../unix_vs_sas_datetime.qmd:
```diff
@@ -0,0 +1,100 @@
+
+    **Detection of date, time, datetime variable from SAS by haven package**
+
+    ![](haven_sas_to_R.png)
```
Could you provide link to source code.  Is that C?

bms63 commented on posts/2024-07-08_unix_vs_sas_date.../unix_vs_sas_datetime.qmd:
```diff
@@ -0,0 +1,100 @@
+
+To address the interoperability issue across programming languages, Dataset-JSON stores numeric date variables as strings in ISO 8601 format.
+Additionally, a `targetdatatype` attribute is included to differentiate between character and numeric dates within the dataset.
+This approach ensures that conversion tools receive sufficient information to handle dates correctly.
```
Can we give a shout out to this R package as well! https://atorus-research.github.io/datasetjson/

bms63 reviewed (changes requested):
Really awesome @cpiraux !!!

bms63 reviewed (commented):
This image is amazing 
![image](https://github.com/user-attachments/assets/4fbea128-ec2e-4838-ba77-f0be62cec12a)

bms63 commented:
I think we just need to get rid of this message from dplyr (@StefanThoma I think this is from session update you did, any thoughts??) 
![image](https://github.com/user-attachments/assets/661dca9c-19af-440a-9724-c3685f0f0f05)

After that I think we are good to go!  @manciniedoardo any last thoughts?

bms63 commented:
oh and we need to fix the spelling workflow

cpiraux commented:
Thanks, everyone, for your review!

I've implemented all the comments. I’ve just updated a paragraph on datasetJSON, as v1.1 is still a draft, by adding a reference to the specification for the latest updates. 

One of the checks is still failing. Do you know what to do to make this check pass?

bms63 commented:
umm..im not sure what is going on with the spelling check.  we can let it slide
